webapp/components/Comment.vue

```diff
@@ -23,49 +23,61 @@
           :modalsData="menuModalsData"
           style="float-right"
           :is-owner="isAuthor(author.id)"
+          @showEditCommentMenu="editCommentMenu"
         />
       </no-ssr>
-      <!-- eslint-disable vue/no-v-html -->
-      <!-- TODO: replace editor content with tiptap render view -->
 
-      <div v-if="isCollapsed" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
-      <div
-        v-show="comment.content !== comment.contentExcerpt"
-        style="text-align: right;  margin-right: 20px; margin-top: -12px;"
-      >
-        <a v-if="isCollapsed" style="padding-left: 40px;" @click="isCollapsed = !isCollapsed">
-          {{ $t('comment.show.more') }}
-        </a>
+      <ds-space margin-bottom="small" />
+      <div v-if="openEditCommentMenu">
+        <hc-edit-comment-form
+          :comment="comment"
+          :post="post"
+          @showEditCommentMenu="editCommentMenu"
+        />
       </div>
-      <div v-if="!isCollapsed" v-html="comment.content" style="padding-left: 40px;" />
-      <div style="text-align: right;  margin-right: 20px; margin-top: -12px;">
-        <a v-if="!isCollapsed" @click="isCollapsed = !isCollapsed" style="padding-left: 40px; ">
-          {{ $t('comment.show.less') }}
-        </a>
+      <div v-show="!openEditCommentMenu">
+        <div v-if="isCollapsed" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-11T10:40:54Z" title="Tuesday, June 11th 2019, 12:40:54 pm +02:00">Jun 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -17,12 +17,20 @@
         :callbacks="{ confirm: deleteCommentCallback, cancel: null }"
         style="float-right"
         :is-owner="isAuthor(author.id)"
+        v-on:showEditCommentMenu="editCommentMenu"
       />
     </no-ssr>
     <!-- eslint-disable vue/no-v-html -->
     <!-- TODO: replace editor content with tiptap render view -->
-    <ds-space margin-bottom="small" />
-    <div style="padding-left: 40px;" v-html="comment.contentExcerpt" />
+    <ds-space margin-bottom="small"/>
+    <div v-if="openEditCommentMenu">
+      <hc-edit-comment-form
+        v-bind:comment="comment"
+        v-bind:post="post"
+        v-on:showEditCommentMenu="editCommentMenu"
```


Not sure if you want to change, or not, and maybe you already know this, but in case you don't, you can write these
```
:comment="comment"
:post="post"
@showEditCommentMenu="editCommentMenu"
```